api/src/client/ApiFormationClient.test.ts

```diff
@@ -593,7 +603,7 @@ describe("ApiFormationClient", () => {
 
             BusinessInformation: {
               CompanyOrigin: "Foreign",
-              PracticesLaw: false,
+              PracticesLaw: "Yes",
```

Comment:
Why do we expect this to be "Yes"? I don't see that anywhere in the setup. Shouldn't this come from setting it to `true` somewhere in the form data during setup?